lib/cartopy/mpl/geoaxes.py

```diff
@@ -392,23 +392,27 @@ class GeoAxes(matplotlib.axes.Axes):
     """
     name = 'cartopy.geoaxes'
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, projection=None, **kwargs):
         """
         Create a GeoAxes object using standard matplotlib
         :class:`~matplotlib.axes.Axes` args and kwargs.
 
         Parameters
         ----------
-        map_projection: optional
-            The target :class:`~cartopy.crs.Projection` of this Axes object.
-
-
-        All other args and keywords are passed through to
-        :class:`matplotlib.axes.Axes`.
-
-        """
-        self.projection = kwargs.pop('map_projection')
-        """The :class:`cartopy.crs.Projection` of this GeoAxes."""
+        projection : cartopy.crs.Projection, required
```

Comment:
`projection` has a default argument so is still optional:
```suggestion
        projection : cartopy.crs.Projection, optional
```

Reply:
Yep, fixed that.